src/Castle.Core/Castle.Core-VS2017.csproj

```diff
@@ -1,42 +1,49 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
Tried to fix this, loaded editorconfig but I think resharper might be overriding this.